BOSS_DEPLOY_DOC.md

```diff
@@ -1,13 +1,32 @@
 # Instructions for Deploying to the Boss
 
+### The main git branch is ndmaster 
+
 # If Your First Time Deploying
 
-Install Node.js >= v5.9.0
+Install Node.js >= v10.23.X
 
 ```shell
 # Install dependencies
 npm i
 ```
+If you had used a different version of Node.js you may want to clear the installed modules before installing them.
+```shell
+#Clear and Install dependencies
+npm ci
+```
+
+To test locally
+```shell
+npi i
```

Comment:
You might note that this may not be necessary if dependencies are already updated. Also there's a typo, should be `npm`.